src/monster.cpp

```diff
@@ -1216,6 +1216,21 @@ int monster::hit_roll() const {
     return dice(type->melee_skill, 10);
 }
 
+bool monster::has_grab_break_tec() const
+{
+    return false;
+}
+
+bool monster::stability_check( monster *z, int strength) const
```

Comment:
Don't I need the monster to have a pointer so I can call each separate monster's melee dice? Without the pointer how is the function going to know which monster is which? 

And the issue with moving the rng(0,10) is that this only returns a bool, not an int.